static/app/views/projectInstall/createProject.tsx

```diff
@@ -36,6 +41,12 @@ type IssueAlertFragment = Parameters<
 function CreateProject() {
   const api = useApi();
   const organization = useOrganization();
+  const location = useLocation();
+  const gettingStartedWithProjectContext = useContext(GettingStartedWithProjectContext);
+
+  const autoFill =
```

Comment:
since the "getting started" docs and the "creating form" pages are on separate routes, depending on where the user is coming from and the project data saved in context... the default values could be wrong so a workaround is to look for these new URL parameters